src/commander/fetchConfigsCommander.ts

```diff
@@ -0,0 +1,43 @@
+import { Command } from "commander";
+
+import { fetchCommandsConfiguration } from "src/command-configs/fetchCommandsConfiguration";
+import { CommandConfigs } from "src/command-configs/types";
+import { getVariablesOption, variablesExitOverride } from "src/commander/getVariablesOption";
+import { config } from "src/config";
+import { LoggerContext } from "src/logger";
+import { PIPELINE_SCRIPTS_REF } from "src/setup";
+
+type ExtendedFetchCommander = Command & {
+  result?: {
+    scriptsRef: string;
+    docsPath: string;
+    commandConfigs: CommandConfigs;
+  };
+};
+
+export function fetchConfigsCommander(ctx: LoggerContext, repo: string): ExtendedFetchCommander {
+  const { botPullRequestCommentMention } = config;
+  const root = new Command(botPullRequestCommentMention) as ExtendedFetchCommander;
+  let scriptsBranch: string | undefined;
+
+  root.addOption(getVariablesOption());
+
+  root.hook("postAction", async (cmd, actionCmd) => {
```

Comment:
Is there a reason why this is a `postAction` and not simply an `action`?

Reply:
yea, all after whole pipe